### ISSUE TITLE
training on 2 classes

### ISSUE THREAD

**Issue by Jee-King** [sacmehta/ESPNet#36]
Hi,
I'm trying to use this model to segment drivable regions and only have 2 classes.  I did not get good results by changing the model.py as you said in #13. The smallest train loss I got was 0.036. So do I need to change softmax function to sigmoid? Or do I need to make other changes?
Thank  you very much.

**Comment by sacmehta** [comment]
Training loss itself is too small and you should get good results. Did you try to measure dice score or mIOU or visualize results on val/test images?

**Comment by Jee-King** [comment]
Thank you for your reply. It has worked.
